src/sentry/integrations/models/integration_feature.py

```diff
@@ -20,65 +20,68 @@
 from sentry.sentry_apps.models.sentry_app import SentryApp
 
 
-class Feature:
-    API = 0
-    ISSUE_LINK = 1
-    STACKTRACE_LINK = 2
-    EVENT_HOOKS = 3
-    PROJECT_MANAGEMENT = 4
-    INCIDENT_MANAGEMENT = 5
-    FEATURE_FLAG = 6
-    ALERTS = 7
-    RELEASE_MANAGEMENT = 8
-    VISUALIZATION = 9
-    CHAT = 11
-    SESSION_REPLAY = 12
+class Feature(IntEnum):
+    API = 0, "integrations-api"
+    ISSUE_LINK = 1, "integrations-issue-link"
+    STACKTRACE_LINK = 2, "integrations-stacktrace-link"
+    EVENT_HOOKS = (
+        3,
+        "integrations-event-hooks",
+    )
+    PROJECT_MANAGEMENT = 4, "integrations-project-management"
+    INCIDENT_MANAGEMENT = 5, "integrations-incident-management"
+    FEATURE_FLAG = 6, "integrations-feature-flag"
+    ALERTS = 7, "integrations-alert-rule"
+    RELEASE_MANAGEMENT = 8, "integrations-release-management"
+    VISUALIZATION = 9, "integrations-visualization"
+    CHAT = 11, "integrations-chat"
+    SESSION_REPLAY = 12, "integrations-session-replay"
+
+    feature_name: str
+
+    def __new__(cls, value: int, feature_name: str) -> Feature:
+        obj = int.__new__(cls, value)
+        obj._value_ = value
+        obj.feature_name = feature_name
```

Comment:
This implementation allows us to specify enums with both int and string values. Overriding `__new__` facilitates constructing an `int` object, with enum values attached in a similar fashion as Python's std lib implementation for HTTP status codes: https://github.com/python/cpython/blob/main/Lib/http/__init__.py